.github/workflows/ty-ecosystem-report.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
 
           cd ..
 
-          uv tool install "git+https://github.com/astral-sh/ecosystem-analyzer@27dd66d9e397d986ef9c631119ee09556eab8af9"
+          uv tool install "git+https://github.com/astral-sh/ecosystem-analyzer@fc0f612798710b0dd69bb7528bc9b361dc60bd43"
```

Comment:
I only bumped it in `ty-ecosystem-analyzer.yaml`, forgot to bump it here as well.